pyemma/msm/tests/test_amm.py

```diff
@@ -0,0 +1,766 @@
+# This file is part of PyEMMA.
```

Comment:
Can't we reconcile this test with the test_msm or test_oom test? It seems we have a lot of redundant code here.